app/helpers/layout_helper.rb

```diff
@@ -113,7 +113,7 @@ def radio_button_f(f, attr, options = {})
   end
 
   def select_f(f, attr, array, id, method, select_options = {}, html_options = {})
-    array = array.to_a
+    array = array.dup.to_a
```

Comment:
Nitpick, I think duplicating the array (after to_a) may be clearer, as at least we always know that it's duping an array and not whatever array-like object `array` happens to be.